static/sass/components/_charts.scss

```diff
@@ -0,0 +1,33 @@
+// SVG Charts
```

Comment:
Are these styles reusable in other posts, not about grid? If they are specific to a post, I wonder if the file should live somewhere else?